lib/matplotlib/backend_bases.py

```diff
@@ -527,12 +527,12 @@ def draw_image(self, gc, x, y, im):
         """
         raise NotImplementedError
 
-    def option_image_nocomposite(self):
+    def option_combine_images(self):
```

Comment:
This change has been in the PR from the beginning.  

I suppose I should have thought about it more carefully.  (This is my first real PR.)  The only backend that used `option_image_nocomposite` was SVG.  In the SVG backend, the user could not set `option_image_nocomposite`.  The user could only query whether it was `True` or `False`.  Correct me if I am wrong, but I believe it would be extremely rare for anyone to query this property.